more/free-programming-cheatsheets.md

```diff
@@ -82,6 +82,7 @@
 
 * [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
 * [GNU Emacs Reference Card](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - GNU.org (PDF)
+* [Learn Vimscript the Hard Way](https://learnvimscriptthehardway.stevelosh.com) - Steve Losh
```

Comment:
It seems more like a tutorial/book than a cheatsheet due to their size and structure. It have a TOC (read CONTRIBUTING)

Reply:
The question now is that is already listed.

https://github.com/EbookFoundation/free-programming-books/blob/521f2e91d2ade5c7df20c361466988136411afb3/books/free-programming-books-langs.md#L2333

Maybe it is a good oportunity to add author and give credits.

Reply:
Oh, sorry I didn't check it carefully. So I change this to update the author.